recipes/jupyterbook-latex/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set name = "jupyterbook-latex" %}
+{% set version = "0.2.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 8083d7aa71bb0dc31847be4edd2b15ef9d55b56a8cf1b07e49b3b627fb7e49c7
+
+build:
+  number: 0
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - pip
+    - python >=3.6
+  run:
+    - python >=3.6
+    - sphinx >=3,<4
+    - myst-nb
+    - pyyaml
+
+test:
+  requires:
+    - pytest
+  imports:
+    - jupyterbook_latex
+    - jupyterbook_latex.theme
+
+
+about:
+  home: "https://github.com/executablebooks/jupyterbook-latex"
+  license: BSD-3-Clause 
+  license_file: LICENSE
+  summary: 'Jupyterbook-latex: Supporting LaTeX infrastructure for Jupyter Book'
+  dev_url: https://github.com/executablebooks/jupyterbook-latex
+
+extra:
+  recipe-maintainers:
+    - AakashGfude
```

Comment:
Feel free to add @conda-forge/jupyter-book ! I accept co-maintenance on our behalf, as it helps us keep track of the moving pieces :grimacing:

Reply:
Wonderful. Does this look good? https://github.com/conda-forge/staged-recipes/pull/14253/files#diff-e93fc87513ebb07179106157a98cd7368c49e0dc30b0eb79480091676ba66971R48

Reply:
Indeed. And we'd :heart:  to add you as a co-maintainer on `jupyter-book-feedstock`! It's most straightforward if you make the PR [adding yourself there](https://github.com/conda-forge/jupyter-book-feedstock/blob/master/recipe/meta.yaml#L67), pretty easy to do through the UI :grin:, and then fire off a `@conda-forge-admin please rerender` so it updates the README. No build number bump would be required.

Reply:
If you _really_ want to run tests (which is cool with me), then, provided the tags are created in a predictable way, one can _also_ download the github tarball (the bot will also update that).


```yaml
source:
- folder: dist
  # whatever's already there
- folder: src
  url: https://github.com/executablebooks/jupyter-book/archive/v{{ version }}.tar.gz
  sha256: <whatever it is>

build:
  scripts: cd dist && # whatever's already there

test:
  source_files:
    - src/tests
  commands:
    - cd src && pytest # whatever

extras:
  license_file: dist/LICENSE
```

Reply:
Thanks, @bollwyvl for your detailed explanation. But, I think the appropriate thing to do for this PR as per your's and @chrisjsewell's comment, is to do only `pip-check` for now. So, I will just pull everything related to `pytest`.

Reply:
> And we'd ❤️ to add you as a co-maintainer on `jupyter-book-feedstock`! It's most straightforward if you make the PR [adding yourself there](https://github.com/conda-forge/jupyter-book-feedstock/blob/master/recipe/meta.yaml#L67), pretty easy to do through the UI 😁, and then fire off a `@conda-forge-admin please rerender` so it updates the README. No build number bump would be required.

have created a PR here https://github.com/conda-forge/jupyter-book-feedstock/pull/3 .  The rerender worked with that comment. pretty dope setup 😬